CHANGES/7698.bugfix

```diff
@@ -0,0 +1 @@
+Fixed downloads outside working directory during sync.
```

Comment:
A typo (it should be `outside of the`) led me to a longer message to make it more user-friendly :)
What do you think?
Fixed the case when downloads were happening outside of the task working directory during sync.